osu.Game/Users/UserPanel.cs

```diff
@@ -110,6 +107,22 @@ private void load()
             Text = User.Username,
         };
 
+        protected virtual Drawable? CreateBackground() => Background = new UserCoverBackground
```

Comment:
This is so weird. So there's one method called `CreateX` and it's virtual and requires override to do something, then there's three other `Create` methods which are required to be *called* by the implementation to actually do something.

At very least this needs xmldoc. Optimally, they shouldn't use the same naming structure.